src/main/java/com/iexec/common/chain/DealParams.java

```diff
@@ -50,6 +51,10 @@
     @JsonProperty("iexec_result_storage_proxy")
     private String iexecResultStorageProxy;
 
+    // Keys are app secrets indices, values are requester secrets indices
```

Comment:
```suggestion
    /*
    The "iexec_secrets" key -> value map declares which secrets of the requester 
    can be accessed by a targeted TEE application.
    - A value should hold the secret name of a secret belonging to the requester.
    This secret must be previously pushed on the SMS. 
    - A key a associated to a value indicates how the TEE application can access 
    that value.
    
    Example:
    
    - A requester has a secret value "2000-01-01" pushed on the SMS. The name of 
    this secret is "my-birthday".
    - A requester has a secret value "+331234567890" pushed on the SMS. The name 
    of this secret is "my-phone-number".

    The application developer creates a TEE application which allows requesters 
    to :
    1. Share their birthday by affecting their secret to position "2"
    2. Share their phone number by affecting their secret to position "5"

    In order to share birthday and phone number to the TEE application, the 
    requester should declare:
    "iexec_secrets" : { "2": "my-birthday" , "5": "my-phone-number" }
    
    By doing this, the TEE application code will be able to read following 
    environment variables:
    IEXEC_REQUESTER_SECRET_2=2000-01-01
    IEXEC_REQUESTER_SECRET_5=+331234567890
    */
```

Reply:
I agree with your suggestions, plus:
* I will try to add it as a javadoc comment
* Maybe the last part should be documented in `iexec-sms,` `iexec-common` has no knowledge of `IEXEC_REQUESTER_SECRET_` prefix

Reply:
Yes, I agree about responsibility of the last part. We can either :
- Add a note stating env var prefix is fake and just for overall understandability purposes
- Relocate that part to `com.iexec.common.utils.IexecEnvUtils`

Reply:
I think it's a good idea to add `IEXEC_REQUESTER_SECRET_` to `IexecEnvUtils`.

Reply:
I have a question about this.
Do I just add the String definition to `IexecEnvUtils` ?
`public static final String IEXEC_REQUESTER_SECRET_PREFIX = "IEXEC_REQUESTER_SECRET_";`
Do I add `IEXEC_APP_DEVELOPER_SECRET_PREFIX` as well ?

Reply:
We can add both yes. Even though the variables will probably be used only in the SMS, it's a good idea to have them all at the same place.

Reply:
See c642ed8 and 86b25b4

Reply:
Thanks!